### PR TITLE
Condense slot result to 1 message

### PR DESCRIPTION
**Pull request by YeahImRose**
As a note I don't know whether or not this will work, I wrote it on my phone. Hopefully it does work though :)

### CODE REVIEW

**Comment by tekulvw** [comment]
Looks good, haven't personally tested it though

File: cogs/economy.py [hunk 1]
```diff
@@ -407,31 +407,31 @@ def already_in_list(self, accounts, user):
         display_reels += ">" + reels[0][1] + " " + reels[1][1] + " " + reels[2][1] + "\n"
         display_reels += "  " + reels[0][2] + " " + reels[1][2] + " " + reels[2][2] + "\n"
 
-        slotMsg = ""
+        slotMsg = "\n"
```
Comment (Fr6jDJF) [inline cogs/economy.py]: I don't know if the blame is on Discord it self, but starting newlines will get stripped/ignored now days so it seems.
Wrapping some random markup around it for example, allows you to start with newlines.
`~~\n~~`

https://github.com/Twentysix26/Red-DiscordBot/pull/236
economyc

File: cogs/economy.py [hunk 2]
```diff
@@ -407,31 +407,31 @@ def already_in_list(self, accounts, user):
         display_reels += ">" + reels[0][1] + " " + reels[1][1] + " " + reels[2][1] + "\n"
```
Comment (Canule) [inline cogs/economy.py]: economyc

File: cogs/economy.py (hunk 1 above)
Comment (YeahImRose) [reply cogs/economy.py]: That's very strange... I'll make sure to keep it in mind in the future

**Pull request merged**